midend/interpreter.cpp

```diff
@@ -309,6 +309,81 @@ void SymbolicStruct::dbprint(std::ostream& out) const {
     out << " }";
 }
 
+SymbolicHeaderUnion::SymbolicHeaderUnion(const IR::Type_HeaderUnion* type,
```

Comment:
How much of this code is duplicated from SymbolicHeader?

Reply:
This code is based on SymbolicHeader as they are quite similar